src/core/include/openvino/runtime/tensor.hpp

```diff
@@ -37,6 +36,7 @@ class OPENVINO_API Tensor {
 protected:
     std::shared_ptr<InferenceEngine::Blob> _impl;  //!< Shared pointer to internal tensor representation
     std::shared_ptr<void> _so;                     //!< Reference to dynamically loaded library
+    std::shared_ptr<void> _addSo;                     //!< Reference to additional loaded library in virual plugin
```

Comment:
Does hold so library work? tensor is bound to compiledModel, is it no problem if compiledModel is released before tensor?

Reply:
per the test result, hold the dynamic loaded library is enough for correctly handling the tensor lifecycle.